source/_components/image_processing.facebox_face_detect.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Facebox Face Detect"
+description: "Count the number of faces in an image with Facebox."
+date: 2018-05-03 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Image Processing
+featured: false
+ha_release: 0.69
+---
+
+The `facebox_face_detect` image processing component allows you to detect (i.e. count the number of) faces in a camera image using [Facebox](https://machinebox.io/docs/facebox). The state of the entity is the number of faces detected. Facebox runs in a Docker container, and it is recommended that you run this container on a machine with a minimum of 2 GB RAM. To minimise the time for Facebox to detect faces in an image using this component, on your machine with Docker run Facebox with:
+```
+MB_KEY="INSERT-YOUR-KEY-HERE"
+
+docker run -p 8080:8080 -e "MB_KEY=$MB_KEY" -e "MB_FACEBOX_DISABLE_RECOGNITION=true" machinebox/facebox
```

Comment:
We should add a name to the container and `restart` option. Here's the command that I use: 
`sudo docker run --name=facebox --restart=always -p 8080:8080 -e "MB_KEY=$MB_KEY" -d machinebox/facebox`

We can still retain the `-e "MB_FACEBOX_DISABLE_RECOGNITION=true"` flag.

Reply:
Sure theres no harm adding this info, thanks for the suggestion.